packages/omezarr/src/sliceview/slice-renderer.ts

```diff
@@ -144,24 +152,34 @@ export function buildOmeZarrSliceRenderer(
             const contents: Record<string, () => Promise<ReglCacheEntry>> = {};
             for (const key in settings.channels) {
                 contents[key] = () =>
-                    decoder(dataset, toZarrRequest(item, settings.channels[key].index), item.level).then(
+                    decoder(dataset, toZarrRequest(item, settings.channels[key].index), item.level, signal).then(
                         sliceAsTexture,
                     );
             }
             return contents;
         },
         isPrepared,
-        renderItem: (target, item, _, settings, gpuData) => {
+        renderItem: (target, item, dataset, settings, gpuData) => {
             const channels = Object.keys(gpuData).map((key) => ({
                 tex: gpuData[key].texture,
                 gamut: intervalToVec2(settings.channels[key].gamut),
                 rgb: settings.channels[key].rgb,
             }));
-
+            // determine the depth at which to render - the goal is that lower resolution tiles should be rendered further back
+            const wholeLayerResolution = planeSizeInVoxels(
+                settings.plane,
+                dataset.attrs.multiscales[0].axes,
+                item.level,
+            );
+            // this number can be very very large - the goal is to have a number from [0:1]...
+            // ideally, we'd go figure out the size of the same plane of the largest layer, but that is annoying right now
+            const divisor = 10_000_000;
+            const depth = Vec2.scale(wholeLayerResolution ?? [0, 0], 1 / divisor);
```

Comment:
this change renders tiles at a Z-depth that is linearly related to their resolution. My plan here was to stop fiddling around with ordered rendering and just let the depth buffer sort it all out. Theres weird annoying math here - item.level used to be an integer telling us which layer its on, but now its a whole shapedDataset.

This isnt a bad thing, and its harmless, but not really used right now, so maybe I should revert it? thoughts?